src/ontology/mondo-ingest.Makefile

```diff
@@ -315,11 +312,50 @@ signature_reports: $(ALL_MIRROR_SIGNTAURE_REPORTS) $(ALL_COMPONENT_SIGNTAURE_REP
 #############################
 #### Lexical matching #######
 #############################
-
 tmp/merged.db: tmp/merged.owl
 	semsql make $@
 
 mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-all-lexical.py 
 	python $^ run tmp/merged.db -c metadata/mondo.sssom.config.yml -r config/mondo-match-rules.yaml -o $@
 
 lexical_matches: mappings/mondo-sources-all-lexical.sssom.tsv
+
+#############################
+###### Slurp pipeline #######
+#############################
+# TODO: (a) Move this to Makefile, or (b) refactor this away.
+.PHONY: component-download-mondo.owl
+component-download-mondo.owl: | $(TMPDIR)
+	if [ $(MIR) = true ] && [ $(COMP) = true ]; then $(ROBOT) merge -I http://purl.obolibrary.org/obo/mondo.owl \
+	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $(TMPDIR)/$@.owl; fi
+
+# TODO: (a) Move this to Makefile, or (b) refactor this away, or (c) DELETE this goal.
+# ...I think (c) is most likely, as `reports/mirror_signature-%.tsv` needs `component-download-%.owl`.
+# ...`reports/component_signature-%.tsv` is what needs `components/%.owl`, but that shouldn't be needed here.
+# $(COMPONENTSDIR)/mondo.owl: component-download-mondo.owl
+# 	if [ $(COMP) = true ] ; then if cmp -s $(TMPDIR)/component-download-mondo.owl.owl $@ ; then echo "Component identical."; else echo "Component is different, updating." && cp $(TMPDIR)/component-download-mondo.owl.owl $@; fi; fi
+
+# todo: add back prereq when done developing: mirror/%.owl
+# mirror/%.db: mirror/%.owl
+mirror/%.db:
+	@rm .template.db.tmp
+	semsql make $@
+	@rm .template.db.tmp
+
+slurp/:
+	mkdir -p $@
+
+# min-id: the next available Mondo ID
+slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/mirror_signature-mondo.tsv | slurp/
+	python $(SCRIPTSDIR)/migrate.py \
+	--ontology-path $(COMPONENTSDIR)/$*.owl \
+	--sssom-map-path $(TMPDIR)/mondo.sssom.tsv \
+	--onto-config-path metadata/$*.yml \
+	--min-id 123000 \
+	--mondo-terms-path $(REPORTDIR)/mirror_signature-mondo.tsv \
+	--outpath $@
+
+slurp-%:
```

Comment:
@matentzn Just wanted to respond to:
>     3. Indentation is critical with Make targets, your `# TODO: Temporary debugging:` was wrongly indented to the level of the target name, not the body.

I know this is a trivial matter; sorry if trying to close the loop on this is a bit much.

Just for reference, since I no longer needed the comment, the change you made was from this:
```make
slurp-%:
# Temporary debugging
```

to this:
```make
slurp-%:
	# Temporary debugging
```

I assume you made this change because you thought either (a) my intention was for the comment to print to the console, or (b) this would cause a syntactical issue.

Regarding (b), I didn't run into any issues with not indenting. It seems to be syntactically correct.
Regarding (a), it didn't actually matter to me if it printed to the console or not. Sometimes I do have a preference. But usually if I want a comment printed to the console, I use `@echo`.

Honestly the main, semi-trivial, reason for my lack of indent in these temporary comment cases is syntax highlighting in PyCharm. Helps me see it so that I can remove it later:
<img width="239" alt="Screen Shot 2022-08-31 at 9 50 53 AM" src="https://user-images.githubusercontent.com/13045020/187700445-c525161e-daac-4d79-827c-64e32a9e6fab.png">

<img width="266" alt="Screen Shot 2022-08-31 at 9 51 25 AM" src="https://user-images.githubusercontent.com/13045020/187700448-bbb3298c-badc-4c27-895d-913363b69110.png">

Reply:
Ok, just be a bit of caution, for example:

```
l1:
	# Comment
	echo "hi"

l2:
# Comment
	echo "hi 2"

l3:
	echo "hi 3.1"
# Comment
	echo "hi 3.2"
```

You are right, all three work. Scratch that, do what you like!

```

Reply:
Will do.

I wasn't actually sure if `l3` would work. Good to know! Gonna mark this one resolved now.